data/datasets/oa_camel/prepare.py

```diff
@@ -0,0 +1,34 @@
+# This program prepares Camel datasets to the Open Assistant format.
+
+import pandas as pd
+from datasets import load_dataset
+import json
+
+# Load the datasets from Hugging Face
+datasets = [
+  'camel-ai/biology', 'camel-ai/physics', 'camel-ai/chemistry', 'camel-ai/math'
```

Comment:
Can you please explain if CamelAI has given written access to using their repository? Is this dataset for research purposes only or can be used for all purposes?